MobSF/utils.py

```diff
@@ -327,3 +327,16 @@ def isDirExists(dir_path):
 
 def genRandom():
     return ''.join([random.SystemRandom().choice('abcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*(-_=+)') for i in range(50)])
+
+
+def zipdir(path, zip_file):
+    """Zip a directory."""
+    try:
+        print "[INFO] Zipping"
+        # pylint: disable=unused-variable
```

Comment:
Since the `_`to denote unused vars is not included in PEP 8 it's not considered :/
![Uploading Screen Shot 2016-11-09 at 07.59.42.png…]()

I will use `_`+ pylint-ignore to match best practices :)